benchmark/benchmarks.jl

```diff
@@ -44,4 +45,34 @@ for structure in [:nonsymmetric, :symmetric],
             end
         end
     end
+
+    SUITE[:coloring][structure][partition][decompression]["n=$n"]["p=$p"] = bench_col
+    SUITE[:decompress][structure][partition][decompression]["n=$n"]["p=$p"] = bench_dec
+end
+
+for structure in [:nonsymmetric, :symmetric],
+    order in [LargestFirst(), SmallestLast(), IncidenceDegree(), DynamicLargestFirst()],
+    n in [10^3, 10^5],
+    p in [2 / n, 5 / n, 10 / n]
+
+    nb_samples = 5
+    As = [sparse(Symmetric(sprand(StableRNG(i), Bool, n, n, p))) for i in 1:nb_samples]
+    if structure == :symmetric
+        gs = [SMC.AdjacencyGraph(A) for A in As]
+        bench_ord = @benchmarkable begin
+            for g in $gs
+                SMC.vertices(g, $order)
+            end
+        end
+    else
+        gs = [SMC.BipartiteGraph(A) for A in As]
+        bench_ord = @benchmarkable begin
+            for g in $gs
+                SMC.vertices(g, Val(1), $order)
```

Comment:
I think we should split the elapsed time for ordering of the rows and the columns?